### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Gkrumbach07**
## This introduces a breaking change

- Yes
- No

## This should yield a new module release

- Yes
- No

### Description

Updated readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,3 +35,6 @@ The application is developed using flask. Use the following step to setup local
 This project is managed with kustomize.
 Use the following command to deploy on OpenShift cluster.
 `kustomize build manifests/overlays/test | oc apply -f -`
+
+### Latest Updates
+Project is being archived. Read comment [here](https://github.com/thoth-station/core/issues/425#issuecomment-1334012921) for more info on how to use this app.
```
Comment (harshad16) [inline README.md]: It would be great if we copy the details here instead of pointing to the issue.

**Review by harshad16 — CHANGES_REQUESTED** [review]
can have one change

File: README.md (hunk 1 above)
Comment (Gkrumbach07) [reply README.md]: will do. also pre-commit is old and failing in this repo

Comment (harshad16) [reply README.md]: change this to https://github.com/thoth-station/ttm-as-a-service/blob/b62caca5e075bb19b2326b16a7bed9110fd3f5b6/.pre-commit-config.yaml#L37
github url inplace of gitlab

File: README.md [hunk 2]
```diff
@@ -35,3 +35,12 @@ The application is developed using flask. Use the following step to setup local
 This project is managed with kustomize.
 Use the following command to deploy on OpenShift cluster.
 `kustomize build manifests/overlays/test | oc apply -f -`
+
+### Latest Updates
+TTM project is being archived. Latest information on TTM related projects can be found over here:  https://github.com/redhat-et/time-to-merge-tool
```
Comment (harshad16) [inline README.md]:
```suggestion
TTM Github App project will not be actively pursued here and this repo is being archived. Users can use this project going forward if they like to or other options like TTM GitHub action is being implemented and available with the latest information on TTM-related projects can be found over here:  https://github.com/redhat-et/time-to-merge-tool
```

**Review by harshad16 — COMMENTED** [review]
one additional comment

**Review by harshad16 — COMMENTED** [review]
/lgtm

thanks :+1:

**Pull request merged**